numpy/core/src/multiarray/scalartypes.c.src

```diff
@@ -4122,7 +4122,7 @@ initialize_numeric_types(void)
 
     /**begin repeat
      * #NAME= Number, Integer, SignedInteger, UnsignedInteger, Inexact,
-     *        Floating, ComplexFloating, Flexible, Character#
+     *        Floating, ComplexFloating, Character#
```

Comment:
@seberg none of these others are abstract base classes?

Reply:
They are, but I suspect `flexible` is the only one implementing things that cannot be inherited.  Maybe that is the bigger problem really, but I guess this is also OK.